pkg/agent/util/net_linux.go

```diff
@@ -237,3 +239,37 @@ func DeleteOVSPort(brName, portName string) error {
 	cmd := exec.Command("ovs-vsctl", "--if-exists", "del-port", brName, portName)
 	return cmd.Run()
 }
+
+func RenameInterface(from, to string) error {
+	klog.InfoS("Rename interface", "oldName", from, "newName", to)
+	var renameErr error
+	pollErr := wait.Poll(time.Millisecond*100, time.Second, func() (done bool, err error) {
+		renameErr = renameHostInterface(from, to)
+		if renameErr != nil {
+			klog.InfoS("Unable to rename host interface name with error, retrying", "oldName", from, "newName", to, "err", renameErr)
+			return false, nil
+		}
+		return true, nil
+	})
+	if pollErr != nil {
+		return fmt.Errorf("failed to rename host interface name %s to %s", from, to)
+	}
+	return nil
+}
+
+func renameHostInterface(oriName string, newName string) error {
```

Comment:
Why creating a separate private function and add "Host" qualifier? It seems to me the public function and the private doesn't have any difference about whether this is a "host" interface. And perhaps there is no need to extract such private function if it's not for code sharing, the deferred function can be executed when the above anonymous function call ends.

Reply:
These 2 functions is copied from BM/VM implementation, and BM/VM will use different functions to do this work in Linux/Windows.